### ISSUE TITLE
[Python 3.9 compat] type is now subscriptable.

### ISSUE THREAD

**Issue by JulienPalard** [PyCQA/pylint#3950]
Since Python 3.9, `type` is made subscriptable by Objects/abstract.c (python/cpython@48b069a), to allow, for example, `type[int]`.

```python
print(type[int])
```

which give:
```
test.py:10:6: E1136: Value 'type' is unsubscriptable (unsubscriptable-object)
```

while the code works.

Related to #3890.
See also #3882.

**Comment by hippo91** [comment]
@JulienPalard thanks for the report.

**Comment by cdce8p** [comment]
This seems to be fixed with: https://github.com/PyCQA/astroid/pull/885
So non blocker after an `astroid` update 🎉

**Comment by hippo91** [comment]
@cdce8p thanks for this check! I close the issue.
